[BUG] Spark 3.3.x integration tests failed due to missing jars

Last night's rapids_it-3.3.x-SNAPSHOT-dev build failed due to missing jars, e.g.:
```
Caused by: java.lang.ClassNotFoundException: com.nvidia.spark.rapids.tests.datasourcev2.parquet.ArrowColumnarDataSourceV2.DefaultSource
```
and
```
E               pyspark.sql.utils.AnalysisException: Can not load class 'com.nvidia.spark.rapids.tests.udf.hive.EmptyHiveSimpleUDF' when registering the function 'emptysimple', please make sure it is on the classpath
```

--- a/integration_tests/run_pyspark_from_build.sh
+++ b/integration_tests/run_pyspark_from_build.sh
@@ -26,12 +26,7 @@ then
 else
     echo "WILL RUN TESTS WITH SPARK_HOME: ${SPARK_HOME}"
     [[ ! -x "$(command -v zip)" ]] && { echo "fail to find zip command in $PATH"; exit 1; }
-    PY4J_TMP=(${SPARK_HOME}/python/lib/py4j-*-src.zip)
-    PY4J_FILE=${PY4J_TMP[0]}
-    VERSION_STRING=$(
-        PYTHONPATH=${SPARK_HOME}/python:${PY4J_FILE} \
-            python -c 'import pyspark; print(pyspark.__version__)'
-    )
+    VERSION_STRING=`$SPARK_HOME/bin/pyspark --version 2>&1|grep -v Scala|awk '/version\ [0-9.]+/{print $NF}'`
     VERSION_STRING="${VERSION_STRING/-SNAPSHOT/}"
     [[ -z $VERSION_STRING ]] && { echo "Unable to detect the Spark version at $SPARK_HOME"; exit 1; }
     [[ -z $SPARK_SHIM_VER ]] && { SPARK_SHIM_VER="spark${VERSION_STRING//./}"; }

